release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -173,6 +173,13 @@ profiles do in Tuned on {op-system-base-full}.
 [id="ocp-4-6-networking"]
 === Networking
 
+[id="ocp-4-6-ovn-kubernetes-ga"]
+==== OVN-Kubernetes default Pod network provider GA
+
+The OVN-Kubernetes default Pod network provider has reached GA. For more information, including regarding feature parity with OpenShift SDN, refer to xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes default Container Network Interface (CNI) network provider].
```

Comment:
s/has reached/is now
s/regarding/details on

Reply:
@ahardin-rh, I find that challenging because I thought GA is _general availability_, and "is now general availability" sounds odd.